onnx/version_converter/adapters/gemm_7_6.h

```diff
@@ -0,0 +1,46 @@
+// Adapter for Gemm in default domain from version 7 to 6
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+class Gemm_7_6 final : public Adapter {
+  public:
+    explicit Gemm_7_6(): Adapter("Gemm", OpSetID(7), OpSetID(6)) {}
+
+    void adapt_gemm_7_6(std::shared_ptr<Graph> graph, Node* node) const {
+      const ArrayRef<Value*>& inputs = node->inputs();
+      ONNX_ASSERTM(inputs.size() == 3, "3 Inputs must be provided to Gemm");
+      ONNX_ASSERTM(inputs[0]->has_sizes(), "Insufficient shape information for 1st input");
+      const auto& A_shape = inputs[0]->sizes();
+      ONNX_ASSERTM(inputs[1]->has_sizes(), "Insufficient shape information for 2nd input");
+      const auto& B_shape = inputs[1]->sizes();
+      // Determine if C is broadcastable
+      ONNX_ASSERTM(inputs[2]->has_sizes(), "Insufficient shape information for 3rd input");
+      const auto& C_shape = inputs[2]->sizes();
+      // Create (M, N) to input to numpy_unibroadcastable
```

Comment:
First make sure A, B, and C's sizes are ints (dims not param).

Then check whether C needs broadcasting or not.